### PR TITLE
Cleanup unused policy rendering code

### PR DESCRIPTION
**Pull request by fasaxc**
Remove unhittable policy rendering code and clean up naming/interfaces.

### CODE REVIEW

File: rules/endpoints_test.go [hunk 1]
```diff
@@ -288,10 +226,10 @@ var _ = Describe("Endpoints", func() {
 
 					{Action: ClearMarkAction{Mark: 0x8}},
 
-					{Comment: "Start of tier tier2",
+					{Comment: "Start of policies",
 						Action: ClearMarkAction{Mark: 0x10}},
 					{Match: Match().MarkClear(0x10),
-						Action: JumpAction{Target: "cali-po-tier2/c"}},
+						Action: JumpAction{Target: "cali-po-/c"}},
```
Comment (neiljerram) [inline rules/endpoints_test.go]: That looks a little odd now; should it be simplified to cali-po-<name> ?

File: rules/policy.go [hunk 2]
```diff
@@ -158,9 +158,9 @@ func (r *DefaultRuleRenderer) CalculateActions(match iptables.MatchCriteria, pRu
 		mark = r.IptablesMarkAccept
 		actions = append(actions, iptables.ReturnAction{})
 	case "next-tier":
```
Comment (neiljerram) [inline rules/policy.go]: Add a case to allow "pass" as well, as a synonym for "next-tier"?

File: rules/endpoints_test.go (hunk 1 above)
Comment (fasaxc) [inline rules/endpoints_test.go]: The leading slash distinguishes between policies and profiles with the same name.  Was just trying to keep as much room for the name.

Comment (neiljerram) [inline rules/endpoints_test.go]: Ah, OK.

**Pull request merged**